README.rst

```diff
@@ -39,6 +39,81 @@ Checkout the `documentation <sphinx-gallery.github.io>`_ for introductions on ho
    :width: 80%
    :alt: A demo of a gallery generated by Sphinx-Gallery
 
+Quickstart
+==========
+
+Sphinx-Gallery can be used to generate an example gallery from `.py` files
```

Comment:
reST single backtick is autolink, double-backticks are code. You want code here I think
```suggestion
Sphinx-Gallery can be used to generate an example gallery from ``.py`` files
```